src/ch05-01-language-basics.md

```diff
@@ -4,12 +4,18 @@
 
 These values are mostly similar to JSON:
 
-| Type | Description | Example
-|---|---|---|
-| integer | Whole number | 1 |
-| float | Floating point number | 1.054 |
-| string | UTF-8 string | "hello!" |
-| path | File or url | ./default.nix |
+| Type          | Description           | Example
+|---------------|-----------------------|------------------|
+| integer       | Whole number          | 1                |
+| float         | Floating point number | 1.054            |
+| string        | UTF-8 string          | "hello!"         |
+| path          | File or url           | ./default.nix    |
+
+And these are the collection types:
+| Type          | Description              | Example
+|---------------|--------------------------|------------------|
+| list          | Multi-type list          | ["hello" 1]      |
+| attribute set | Key value structure      | {key = "value";} |
```

Comment:
nit: but I think whitespace makes it a bit easier to read, also we may want to backquotes so that it's monospaced
```suggestion
| list          | Multi-type list          | `[ "hello" 1 ]`      |
| attribute set | Key value structure      | `{ key = "value"; }` |
```